plugins/outputs/groundwork/README.md

```diff
@@ -32,7 +32,18 @@ This plugin writes to a [GroundWork Monitor][1] instance. Plugin only supports G
 
 * service  - to define the name of the service you want to monitor.
 * status   - to define the status of the service.
+   * Valid Values are:
+   * Ok = "SERVICE_OK"
+   * Unscheduled Warning = "SERVICE_UNSCHEDULED_WARNING"
+   * Unscheduled Critical = "SERVICE_UNSCHEDULED_CRITICAL"
+   * Pending = "SERVICE_PENDING"
+   * Scheduled Critical = "SERVICE_SCHEDULED_CRITICAL"
+   * Unknown = "SERVICE_UNKNOWN"
```

Comment:
Linter is complaining about `Unordered list indentation [Expected: 2; Actual: 3]`

```suggestion
  * Valid Values are:
  * Ok = "SERVICE_OK"
  * Unscheduled Warning = "SERVICE_UNSCHEDULED_WARNING"
  * Unscheduled Critical = "SERVICE_UNSCHEDULED_CRITICAL"
  * Pending = "SERVICE_PENDING"
  * Scheduled Critical = "SERVICE_SCHEDULED_CRITICAL"
  * Unknown = "SERVICE_UNKNOWN"
```